spec/mailers/previews/report_mailer_preview.rb

```diff
@@ -1,3 +1,4 @@
+require 'reports/monthly_key_metrics_report'
```

Comment:
Apologies if I'm overlooking the obvious, but I don't see where we use this here. Do we actually need this `require`?

Reply:
Great catch - that was left over from my troubleshooting work…